### ISSUE TITLE
"feature" where can i get item x

### ISSUE THREAD

**Issue by BlackWolfDEsign** [jansepke/anno-toolkit#78]
Hi maybe is it possible to display the vendor of specific item?

**Comment by jansepke** [comment]
yes, that should be possible, as the necessary data is available, but currently I don't have the time to implement this. But I would support a PR if someone wants to add it

**Comment by Oskar1504** [comment]
[PR](https://github.com/jansepke/anno-toolkit/pull/112) adds this feature.
